sdk/js/src/utils/repairVaa.test.ts

```diff
@@ -0,0 +1,82 @@
+  expect(() => {
+    repairVaa(vaa, guardianSetData);
+  }).toThrow("There are not enough valid signatures to repair.");
+});
+
+test("number of vaa signatures does not match parse signatures", () => {
+  // Attempting to parse vaa with fake # of signatures. error thrown by parseVaa function
+  const vaa =
+    "01000000011100249871e73fc73a19ed47d9d6c39551b1b2e92f5fc76546b4c2d790845563b3aa6b413553d633fb2ede0b6491d6b8b55738bb89a648e01077eb29ad452bc32de6000295f8bfb0394d23826fecba81f73af18278dac42b665efb148e0a8b380532f2403a6e776851d6f4d2c0f6ae61f8efa699c856802f707a64b4f9e320f4483017cc0003397afeaba4e9c064fd99c79caac2bff7c9c147cde9286b26b242afe0367bd9fe227b345ad0f7d51cbcfd63b5d8593ce65918bfe8c1794b6316869638fc73cf530104df5411869d2e7c5de5de5f1e199db371ac40e7e8adb00cc8a2ed999fe2d02b75278fe8442306203206ec45efdffa5208fe70a3eeaa8125b4fb435aac103037860106106e85c3fcb8ebf231e176e8854f9b04a6ae777cbbbda3f0f1bcd1e08cecc9da420c4af21145972d7d538e28cdd957dfe6ff048d6a403393ed5ca44dd9f1180b000730c77ea4586080203329cda5b74022f1f53bc56fad600cb89f73c75457fd963271d9ff0425b7cdff2e013598a58ac413163b0e878bf01979104d67d9a4e85e0a010948f3dd5a01023936384410fd31c3e58c6b808dfe0826b7eb6af695884d20e95e47bbe093af574ed2640cfe32b6171a7957cede3bbf9d8e5374547f49ca00cbc8010c1b194773d701095798252c356ee224e8ba188b8f8d09975687efc3b376b2cf496e07c267c6b54f2ce98cc262c4c3bc28229ff1e257afe7d78cc9569a660eb04c000d61e3e135c605ba4b3d39b178f0dfc0db6497291b29c97afed6f6feedcd7fbe052795861e4ef05f7d715f313c4cafca443ad1ca69d61aecde7b1aee9e3cf520f2000e617900cb13db3eb68bdbbc159bcced7a064cdbcc7253d10a1a6ea5cd6f5d9fa87b2b1f7d474180da2534db5f478607ad3c82d0b7cc30734f3a480e77a80d13e6010fb84790f4637d4f78612f7b734daa1a085fc21ebcca4f89279a26a12e8836c6a7437342392a034dda85d3a0330f425207132df39d49688a05e0f316d8cc4c721e0010c77c0979a9034a129ed211e5c720a1df2d024cc7d45ae174f779127a089900142b144b5ce6513d19df1e1528d574a528300c06229abb1ec50fc58c0fac7dc7400012090fcaf1f83cee46756e3a9c370fe2a6c7c421bc7cfa5920c044d36b6c2130204e819edd895249c2686d3330565c583f53b89cfdc4275fd56baa570bb442fc4701637431d7a33b000000020000000000000000000000003ee18b2214aff97000d974cf647e7c347e8fa585000000000001685e010100000000000000000000000000000000000000000000000000000001d5a977820000000000000000000000001f9840a85d5af5bf1d1762f925bdaddc4201f984000200000000000000000000000027ec3715bc5e419df98689bd301c17661c73c2c300040000000000000000000000000000000000000000000000000000000000000000";
+  expect(() => {
+    repairVaa(vaa, guardianSetData);
+  }).toThrow("Attempt to access memory outside buffer bounds");
+});
+
+test("Use current guardian set", async () => {
```

Comment:
Can we do `test.skip` here and the test below so we don't hit the public ankr RPC in the CI tests?